elfi/methods/parameter_inference.py

```diff
@@ -3,18 +3,19 @@
 __all__ = ['Rejection', 'SMC', 'BayesianOptimization', 'BOLFI']
 
 import logging
+from collections import OrderedDict
 from math import ceil
 
-import matplotlib.pyplot as plt
 import numpy as np
+from IPython import display
 
 import elfi.client
 import elfi.methods.mcmc as mcmc
 import elfi.model.augmenter as augmenter
 import elfi.visualization.interactive as visin
 import elfi.visualization.visualization as vis
 from elfi.loader import get_sub_seed
-from elfi.methods.bo.acquisition import LCBSC
+from elfi.methods.bo.acquisition import LCBSC, MaxVar, RandMaxVar
```

Comment:
In this case, the first line of the initialisation corresponding to e.g. Line 296 (this PR's source) would contain an amount of letters which is close to 80. It would make the currently applied column-based attribute listing not suitable thus the current readability style would suffer.

Reply:
Our limit for the line length is "the extended PEP8" corresponding to 99 characters + newline.

Though actually I guess you don't need to import other acq. methods but the default (which ever we set it to be).